packages/blade/src/utils/useBreakpoint/useBreakpoint.web.test.ts

```diff
@@ -20,7 +20,7 @@ describe('useBreakpoint', () => {
       // eslint-disable-next-line @typescript-eslint/no-invalid-void-type
       .fn<void, ((arg: { media: string }) => void)[]>()
       .mockImplementation((_, callback: (arg: { media: string }) => void) => {
-        callback({ media: 'screen and (min-width: 769px) and (max-width: 1024px)' });
+        callback({ media: 'screen and (min-width: 1024px) and (max-width: 1199px)' });
```

Comment:
can you add assertion where maxWidth will be undefined. let's say for `screenwidth: 1368px`

Reply:
https://github.com/razorpay/blade/blob/e91fa44df318aa7ada9440986c362e5ec21d69f4/packages/blade/src/utils/getMediaQuery/getMediaQuery.test.ts#L5

`getMediaQuery` function's test covers more granular edge cases

Reply:
Also btw the media query in this `useBreakpointValue` test has to be same as media query that we add on breakpoint otherwise it doesn't work 🤔